WardReports/Generate_weekly_ward_report_scripts.py

```diff
@@ -0,0 +1,186 @@
+import pandas as pd
+import numpy as np
+
+import datetime as dt
+import math
+
+import civis
+import os
+
+#########################################################
+#Pulling in data for email (to be transferred to a different script later)
+
+client = civis.APIClient()
```

Comment:
once all the helpers are in a helper file, lets puts all the main code in a `main()` function and then call main at the very bottom